sbi/inference/__init__.py

```diff
@@ -1,29 +1,12 @@
-from typing import (
-    Any,
-    Callable,
-    Dict,
-    List,
-    Optional,
-    Sequence,
-    Tuple,
-    TypeVar,
-    Union,
-    cast,
-)
-
-from sbi.inference.abc.mcabc import MCABC
-from sbi.inference.abc.smcabc import SMCABC
```

Comment:
these should stay, no?

Reply:
I added the same hierarchy for them as for SNPE i.e. the are imported from `sbi.inference.abc` (ruff moved it to the top because it starts with a)